### PR TITLE
adding kubecon recap blog

### PR DESCRIPTION
**Pull request by Anita-ihuman**

### CODE REVIEW

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Another neat find was the Kubernetes Book Club where people meet up virtually every week and discuss the next chapter in a Kubernetes related book. Their session in the event wasn’t a regular club meeting, but rather an introduction to the club. Additionally, two book authors, Mauricio Salatino and Chad Crowell, were present in person. They talked about their books and their work in the cloud native ecosystem, and even gave out signed copies. We didn’t get a book because so many people wanted them, the books ran out before everybody could get one. Maybe next year!
+Our favorite talk, also on Thursday, was “[Kubernetes Controllers in Rust: Fast, Safe, Sane](https://kccnceu2024.sched.com/event/1YeOR/kubernetes-controllers-in-rust-fast-safe-sane-matei-david-buoyant)” by Matei David, also from Buoyant where we found about Kubert, which might be really useful for some of our use cases! Kubert is built on top of kube-rs, a Rust crate for Kubernetes API. It provides abstractions and batteries for building operators/controllers and can be compared to the Operator Framework.
```
Comment (t4lz) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md:44]:
```suggestion
Our favorite talk, also on Thursday, was “[Kubernetes Controllers in Rust: Fast, Safe, Sane](https://kccnceu2024.sched.com/event/1YeOR/kubernetes-controllers-in-rust-fast-safe-sane-matei-david-buoyant)” by Matei David, also from Buoyant where we found about Kubert, which might be really useful for some of our use cases! Kubert is built on top of kube-rs, a Rust crate for interacting with the Kubernetes API. It provides abstractions and batteries for building operators/controllers and can be compared to the Operator Framework.
```
> a Rust crate for Kubernetes API

"Kubernetes API" isn't a name (only "Kubernetes" is), so AFAICT it needs a "the".
And just "for the Kubernetes API" did not sound accurate to me so I suggest "for interacting with the kubernetes API".

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+We had a fun surprise on the last day of the conference when a team that uses mirrord came up to us with an informal bug report (the bug wasn’t the fun part). There’s nothing better for us than actually meeting the people who use the tool we built on a daily basis.
```
Comment (t4lz) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md:45]:
```suggestion
We had a fun surprise on the last day of the conference when a team that uses mirrord came up to us with an informal bug report (the bug wasn’t the fun part). There’s nothing better for us than actually meeting the people who use the tool we build on a daily basis.
```
I think this
> we built on a daily basis

should be
> we build on a daily basis

right?

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Closing Thoughts
+
+To wrap things up: our first KubeCon was a blast. We had a chance to present mirrord, geek out about it with a great crowd, and meet our community face to face. We’re already waiting for KubeCon NA (at Salt Lake City, cloud-native capital of the western frontier!), and we might even do a booth this time. Looking forward to seeing everyone there!
```
Comment (t4lz) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md:49]:
```suggestion
To wrap things up: our first KubeCon was a blast. We had a chance to present the unique technical details of the implementation of mirrord, geek out about it with a great crowd, and meet our community face to face. We’re already waiting for KubeCon NA (at Salt Lake City, cloud-native capital of the western frontier!), and we might even do a booth this time. Looking forward to seeing everyone there!
```
We didn't present the tool itself, but rather its special implementation details.

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+date: 2024-04-03T0:00:00+00:00
+lastmod: 2024-04-03T0:00:00+00:00
+draft: false
+weight: 50
+contributors: ["Aviram Hassan", "Tal Zwick"]
+---
+
+Last week marked a huge milestone for us as we attended our first KubeCon! This was KubeCon EU which took place in Paris, city of lights and cloud orchestration. Below is a summary of our collective impressions and insights.
+
+# Co-Located Sessions and Surprising Highlights
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:20]:
```suggestion
## Co-Located Sessions and Surprising Highlights
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  - AI/ML
+  - platform-engineering 
+date: 2024-04-03T0:00:00+00:00
+lastmod: 2024-04-03T0:00:00+00:00
+draft: false
+weight: 50
+contributors: ["Aviram Hassan", "Tal Zwick"]
+---
+
+Last week marked a huge milestone for us as we attended our first KubeCon! This was KubeCon EU which took place in Paris, city of lights and cloud orchestration. Below is a summary of our collective impressions and insights.
```
[the remark this answers is not in the document]
Comment (eyalb181) [reply content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md]: The "the" isn't necessary, matter of taste, leaving it up to Anita

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+Another neat find was the Kubernetes Book Club where people meet up virtually every week and discuss the next chapter in a Kubernetes related book. Their session in the event wasn’t a regular club meeting, but rather an introduction to the club. Additionally, two book authors, Mauricio Salatino and Chad Crowell, were present in person. They talked about their books and their work in the cloud native ecosystem, and even gave out signed copies. We didn’t get a book because so many people wanted them, the books ran out before everybody could get one. Maybe next year!
```
[the remark this answers is not in the document]
Comment (eyalb181) [reply content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md]: I think 2/3 Rust bits is good enough

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md (hunk 2 above)
Comment (eyalb181) [reply content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md]: The meaning here is that they use it on a daily basis, not that we build it on a daily basis (even though we do). So I think "built" removes that ambiguity.

Comment (Anita-ihuman) [reply content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/_index.md]: I think its ok as it is.

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+title: "KubeCon 2024 Takeaways: What We Learned Without Getting Our Heads Spun"
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:2]:
```suggestion
title: "KubeCon 2024 Takeaways"
```

The subtitle is a bit weird, I'd stay with just the first part

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+description: "A summary of our collective impressions and insights from KubeCon Europe  2024"
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:3]:
```suggestion
description: "A summary of our collective impressions and insights from KubeCon Europe 2024"
```

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+lead: "A summary of our collective impressions and insights from KubeCon Europe  2024"
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:4]:
```suggestion
lead: "A summary of our collective impressions and insights from KubeCon Europe 2024"
```

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  - platform-engineering 
+date: 2024-04-03T0:00:00+00:00
+lastmod: 2024-04-03T0:00:00+00:00
+draft: false
+weight: 50
+images: ["thumbnail.png"]
+contributors: ["Aviram Hassan", "Tal Zwick"]
+---
+
+March marked a huge milestone for us as we attended our first KubeCon! This was KubeCon Europe, which took place in Paris, a city of lights and cloud orchestration. Below is a summary of our collective impressions and insights.
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:19]:
```suggestion
Last month marked a huge milestone for us as we attended our first KubeCon! This was KubeCon Europe, which took place in Paris, city of lights and cloud orchestration. Below is a summary of our collective impressions and insights.
```

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+## Co-Located Sessions and Surprising Highlights
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:21]:
```suggestion
## Co-Located Events and Surprising Highlights
```

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+{{<figure src="first-image.png" alt="KubeCon 2024 Takeaways " height="100%" width="100%">}}
+
+
+## The Main Event™
+
+The next day the main KubeCon event officially started, and it was PACKED. After two years of mostly working from our homes, we finally got to say hi to friends from all over the world who are part of our awesome community. Tal did his talk “[**When They Go High, We Go Low – Hooking Libc Calls to Debug Kubernetes Apps**](https://youtu.be/3OSQdiKTNU8?si=wP3IPbMM2g-GSBl-)”. There was a turnout of about 400 people, which was roughly 10 times what we expected - this was not lost on Tal. Feedback was incredibly positive though, and we lingered in the auditorium long after the talk to answer questions, discuss mirrord, and hand out some quality swag.
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:30]: The bold part doesn't render well - check the preview

Comment (eyalb181) [reply content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:30]: (Same with the Rustvolution and Kubernetes Controllers link)

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+We had a fun surprise on the last day of the conference when a team that uses mirrord came up to us with an informal bug report (the bug wasn’t the fun part). There’s nothing better for us than actually meeting the people who use the tool we built on a daily basis.
+
+## Closing Thoughts
+
+To wrap things up: our first KubeCon was a blast. We had a chance to present the unique technical details of the implementation of mirrord, geek out about it with a great crowd, and meet our community face to face. We’re already waiting for KubeCon NA (at Salt Lake City, cloud-native capital of the western frontier!), and we might even do a booth this time. Looking forward to seeing everyone there!
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:51]:
```suggestion
To wrap things up: our first KubeCon was a blast. We had a chance to present the unique technical details of the implementation of mirrord, geek out about it with a great crowd, and meet our community face to face. We’re already waiting for KubeCon North America (at Salt Lake City, cloud-native capital of the western frontier!), and we might even do a booth this time. Looking forward to seeing everyone there!
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+weight: 50
+images: ["thumbnail.png"]
+contributors: ["Aviram Hassan", "Tal Zwick"]
+---
+
+Last month marked a huge milestone for us as we attended our first KubeCon! This was KubeCon Europe, which took place in Paris, city of lights and cloud orchestration. Below is a summary of our collective impressions and insights.
+
+## Co-Located Events and Surprising Highlights
+
+We arrived a day before KubeCon proper started so we could attend the co-located events. We ended up mostly going to AppDeveloperCon sessions, and it was surprisingly crowded. Our highlight from the first day was Daniel Bryant’s excellent talk “**Testing Cloud Apps: Mocks vs. Service Virtualization vs. Remocal Tools**” ([slides](https://speakerdeck.com/danielbryantuk/testing-cloud-apps-mocks-vs-service-virtualization-vs-remocal-tools), [video](https://www.youtube.com/watch?v=3tP31j_T6IE)). Daniel covered our favorite pain - being unable to work against a real environment and iterate with it as part of the development workflow. We were pleased to see a very friendly shoutout to mirrord and our to talk later in the convention!
```
Comment (eyalb181) [inline content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:23]: To keep things consistent, lets just have all talk titles link directly to the video. No need to link to the slides here.

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md (hunk 14 above)
Comment (Anita-ihuman) [reply content/en/blog/kubecon-2024-takeaways-what-we-learned-without-getting-our-heads-spun/index.md:23]: I agree

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**